Lib/difflib.py

```diff
@@ -2036,6 +2036,10 @@ def make_table(self,fromlines,tolines,fromdesc='',todesc='',context=False,
                 s.append( fmt % (next_id[i],next_href[i],fromlist[i],
                                            next_href[i],tolist[i]))
         if fromdesc or todesc:
+            fromdesc = fromdesc.replace("&", "&amp;").replace(">", "&gt;") \
```

Comment:
Why not use `html.escape()`?

Reply:
I thought about the same but `text` which was encoded was using the logic [inlined](https://github.com/python/cpython/blob/c465682718f15cd3deb6b37db5fb607718ac64ed/Lib/difflib.py#L1915) and there was some discussion in the issue about just inlining the logic to avoid dependency at https://bugs.python.org/issue914575#msg45518 which I realize now is xml.sax and not html. 

html module was also importing entities at https://github.com/python/cpython/blob/c465682718f15cd3deb6b37db5fb607718ac64ed/Lib/html/__init__.py#L6 that seemed little import heavy for me. But if it's okay to import html then I think it's good to use `html.escape` across the module to avoid duplicate code.

Reply:
Well, for a bug fix we can keep an inlined code and consider using `html.escape()` in a separate issue.

Reply:
Sure, thanks for the review.